services/provider/server/server.go

```diff
@@ -1813,6 +1826,42 @@ func (s *OCSProviderServer) appendVolumeReplicationClassKubeResources(
 	return kubeResources, nil
 }
 
+func (s *OCSProviderServer) appendVolumeGroupReplicationClassKubeResources(
+	ctx context.Context,
+	kubeResources []client.Object,
+	consumer *ocsv1alpha1.StorageConsumer,
+	consumerConfig util.StorageConsumerResources,
+	storageCluster *ocsv1.StorageCluster,
+	rbdStorageId string,
+	remoteRbdStorageId string,
+) ([]client.Object, error) {
+	if mirrorEnabled, err := s.isConsumerMirrorEnabled(ctx, consumer); err != nil {
+		return kubeResources, err
```

Comment:
```suggestion
		return nil, err
```
no need for caller to second guess